socrata-http-jetty/src/main/scala/com/socrata/http/server/AbstractSocrataServerJetty.scala

```diff
@@ -325,7 +321,6 @@ object AbstractSocrataServerJetty {
     override def withExtraHandlers(h: List[Handler => Handler]) = copy(extraHandlers = h)
     override def withErrorHandler(h: Option[HttpRequest => HttpResponse]) = copy(errorHandler = h)
     override def withPoolOptions(poolOpt: Pool.Options) = copy(poolOptions = poolOpt)
-    override def withIdleTimeout(it: Int) = copy(idleTimeout = it)
```

Comment:
this does seem like it is part of the public API though, are we worried about this breaking things?